gcs_to_bq/main.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import json
+import logging
+from common.census_to_bq import write_state_names_to_bq
+
+
+_HOUSEHOLD_INCOME = 'HOUSEHOLD_INCOME'
+_STATE_NAMES = 'STATE_NAMES'
```

Comment:
I had a similar comment on Jennie's PR but now that we have them in two places we should consider creating a `constants.py` module for these types of values.

Reply:
See other comment about sharing code between functions.